pkg/machines/include/Orbitron700.ttf

```diff
@@ -0,0 +1 @@
+../../../bower_components/no-vnc/include/Orbitron700.ttf
```

Comment:
Shipping 20 static symlinks to various files in noVNC seems really ugly and hard to maintain - tests will very likely not detect if a newer upstream version ships a new or renamed static file which would need to be updated here. Please rather iterate over bower_components/no-vnc/include/ in the Makefile and create the symlinks dynamically.